docs/developer_guide.md

```diff
@@ -36,7 +36,7 @@ an Administrator Powershell window:
 
 ## Building the repository
 
-Generally you will want to set your configuration to **Debug**, **x86**, and 
+Generally you will want to set your configuration to **Debug**, **x64**, and 
```

Comment:
This the better inner loop now?

Reply:
We're making 64-bit the default everywhere so I figure we should get comfortable debugging that config. Plus it means you don't use wow when debugging on your dev box.